internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,443 @@
+	if err != nil {
+		return err
+	}
+	// Getting Kafka Cluster & SR
+	flagsExport := flags{
+		kafkaApiKey:             flagsImp.kafkaApiKey,
+		schemaRegistryApiKey:    flagsImp.schemaRegistryApiKey,
+		schemaRegistryApiSecret: flagsImp.schemaRegistryApiSecret,
+	}
+	details, err := c.getAccountDetails(&flagsExport)
+	if err != nil {
+		return err
+	}
+	for topicName, topicDetails := range spec.Channels {
+		err := c.addChannelToCluster(details, spec, topicName, topicDetails.Bindings.Kafka, flagsImp.overwrite)
+		if err != nil {
+			if err.Error() == parseErrorMessage {
+				log.CliLogger.Info(err)
+			} else {
+				log.CliLogger.Warn(err)
+			}
+		}
+	}
+	output.Printf("AsyncAPI specification imported from \"%s\"\n", args[0])
+	return nil
+}
+
+func fileToSpec(fileName string) (*Spec, error) {
+	asyncSpec, err := os.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+	spec := new(Spec)
+	err = yaml.Unmarshal(asyncSpec, spec)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal YAML file: %v", err)
+	}
+	return spec, nil
+}
+
+func (c *command) addChannelToCluster(details *accountDetails, spec *Spec, topicName string, kafkaBinding kafkaBinding, overwrite bool) error {
+	topicExistsAlready, newTopicCreated, err := c.addTopic(details, topicName, spec.Channels[topicName].Description, kafkaBinding, overwrite)
+	if err != nil {
+		// unable to create kafka topic
+		log.CliLogger.Warn(err)
+	}
+	// If topic exists and overwrite flag is false, move to the next channel in spec
+	if topicExistsAlready && !overwrite {
+		return errors.New(parseErrorMessage)
+	}
+	// Register schema
+	schemaId, err := registerSchema(details, topicName, spec.Components)
+	if err != nil {
+		return err
+	}
+	// Update subject compatibility
+	if spec.Channels[topicName].XMessageCompatibility != "" {
+		if err := updateSubjectCompatibility(details, spec.Channels[topicName].XMessageCompatibility, topicName+"-value"); err != nil {
+			return err
+		}
+	}
+	// Add tags
+	if err := addSchemaTags(details, spec.Components, topicName, schemaId); err != nil {
+		return err
+	}
+	if topicExistsAlready || newTopicCreated {
+		if err := addTopicTags(details, spec.Channels[topicName].Subscribe, topicName); err != nil {
+			return err
+		}
+	}
+	// Add topic description to newly created topic
+	if (topicExistsAlready || newTopicCreated) && spec.Channels[topicName].Description != "" {
+		if err = addTopicDescription(details.srClient, details.srContext, details.clusterId+":"+topicName,
+			spec.Channels[topicName].Description); err != nil {
+			return fmt.Errorf("unable to update topic description: %v", err)
+		}
+	}
+	return nil
+}
+
+func isModifiableConfig(configName string) bool {
+	modifiableConfigs := []string{"delete.retention.ms", "max.compaction.lag.ms", "max.message.bytes", "message.timestamp.difference.max.ms", "message.timestamp.type",
+		"min.compaction.lag.ms", "min.insync.replicas", "retention.bytes", "retention.ms", "segment.bytes", "segment.ms"}
```

Comment:
What if we add more values in the future? The topic API can provide you this list, so let's use that?